src/utils/normalize-mime-parts.ts

```diff
@@ -77,9 +77,19 @@ export function normalizeMimeParts(
 		isDesktop
 	}: { isDesktop?: string; jwtToken?: string; origin?: string }
 ) {
-	const processAttachment = ({ ...attachment }) => {
+	const processAttachment = (
+		{ ...attachment },
+		forcedContentDisposition?: any
+	) => {
 		attachment.messageId = attachment.messageId || message.id;
 		attachment.url = getAttachmentUrl(attachment, { origin, jwtToken });
+
+		if (forcedContentDisposition === 'attachments') {
```

Comment:
this should be added only when original attachment data doesn't have content-disposition header
we shouldn't change value if already present

Reply:
Actually we are doing the same now as well but will add conditional code for betterment